launcher/modManager/cmodlistview_moc.cpp

```diff
@@ -685,7 +718,9 @@ void CModListView::installFiles(QStringList files)
 	{
 		if(filename.endsWith(".zip"))
 			mods.push_back(filename);
-		if(filename.endsWith(".json"))
+		else if(filename.endsWith(".h3m") || filename.endsWith(".h3c") || filename.endsWith(".vmap") || filename.endsWith(".vcmp"))
+			maps.push_back(filename);
+		else if(filename.endsWith(".json"))
```

Comment:
According to Qt docs these checks are case-sensitive. Worked fine for "nice" data from github, but user-provided files might come with any case.